suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -67,7 +67,8 @@ export const XpubScanScreen = ({
     const form = useForm<XpubFormValues>({
         validation: xpubFormValidationSchema,
     });
-    const { handleSubmit, setValue } = form;
+    const { handleSubmit, setValue, watch, reset } = form;
+    const watchXpubAddress = watch('xpubAddress');
```

Comment:
Why do we need this?

Reply:
This is the necessary dependency for `useCallback` mentioned in the other comment https://github.com/trezor/trezor-suite/pull/6428#discussion_r984333568. According to it: setValue changes form state, then submit onXpubFormSubmit is rerendered, it means handleXpubResult is rerendered, setValue changes state...

Reply:
I somehow doesn't understand, from what I see it looks like seems redundant. It is used in that if I think we don't need it here.